av_node_2:
  ros__parameters:

    camera_name: DEV_1AB22C00BBE5

    publishers:
      raw_frame: /arm_camera_2/raw_frame

    services:
      stop_camera: /arm_camera_2/get_cam_state

    frames: 
      camera_link: wrist_camera_link_2
      
    rotation_angle: 180.0

    auto_exposure: Once
    
    resize:
      enable: False

    crop:
      enable: False

